lib/hydra/works/models/concerns/generic_file_behavior.rb

```diff
@@ -0,0 +1,44 @@
+module Hydra::Works
+  module GenericFileBehavior
```

Comment:
If I mix Hydra::Works::GenericWorkBehaviors into ScannedBook, that implies ScannedBook has the Generic behaviors of a Work.  That's not true.  It has the behaviors of a Work, applied to a *specific* implementation.  The only sense in which GenericWorkBehaviors are generic is the sense in which every OOP class is a Generic implementation of its instances.  By that logic, all OOP classes would always include Generic in their name.  GenericUser, GenericCar, GenericComment...